srtcore/epoll.h

```diff
@@ -359,6 +410,26 @@ friend class CRendezvousQueue;
 
    int wait(const int eid, std::set<SRTSOCKET>* readfds, std::set<SRTSOCKET>* writefds, int64_t msTimeOut, std::set<SYSSOCKET>* lrfds, std::set<SYSSOCKET>* lwfds);
 
+   typedef std::map<SRTSOCKET, int> fmap_t;
+
+   int swait(CEPollDesc& d, fmap_t& st, int64_t msTimeOut, bool report_by_exception = true);
+
+   static int ready(const fmap_t& mp, SRTSOCKET sock)
```

Comment:
Utility functions. Will be used in the group code.

Reply:
Add docs (in comments) for the three new functions